### PR TITLE
Update goose images

### PR DESCRIPTION
**Pull request by jmawloo**
Updated Goose Images and Content for both Goose III and Goose V. Content is also updated for Goose IV. Still waiting on some images for Goose IV, and potentially a banner image for Goose V.

### CODE REVIEW

File: routes/flock.json [hunk 1]
```diff
@@ -119,43 +119,282 @@
...
     "url": "goose_iii",
-    "desc": "",
-    "banner": "/images/flock/goose3/torontorender.jpg",
+    "desc": "The driving frontier.",
+    "banner": "/images/flock/goose3/g3-render.png",
     "sections": [
       {
+        "title": "Specifications",
+        "content": [
+          "Our pod design process for Competition III began with the goal of building the fastest pod. Based purely on specific power and energy of lithium ion batteries (and perhaps supercapacitors), it is theoretically possible to achieve speeds above Mach 1 within the 1-mile Hyperloop test track. This pod is designed to achieve 100m/s, and we are gradually working towards higher and higher speeds for future competitions.",
```
Comment (Stephenyhlin) [inline routes/flock.json]: Take out the word "specific"

Comment (Stephenyhlin) [reply routes/flock.json]: Also change "we are gradually working towards higher and higher speeds for future competitions."

to "we are gradually working towards higher speeds for future competitions."

File: routes/flock.json [hunk 2]
```diff
@@ -119,43 +119,282 @@
...
+          "The prototype is about 0.5m (20\") wide, 2.7m (8.9') long, 0.8m (31.5\") tall, around 150 kg in weight, and can theoretically reach speeds at or above Mach 1 (1235km/h, or 343m/s)."
```
Comment (Stephenyhlin) [inline routes/flock.json:131]: Get rid of comma before and

File: routes/flock.json [hunk 3]
```diff
@@ -119,43 +119,282 @@
...
+        "title": "Propulsion",
+        "content": [
+          "The most challenging design yet, our propulsion system relies on a friction drive design to propel the pod forward. Using two rotating elements and a belt drive system, power is transferred from the motor to propel the pod along a surface safely and reliably. The friction drive design serves as a precursor to our Linear Induction Motor (LIM) design in later Goose pods."
+        ],
+        "parallax": "/images/flock/goose3/friction_rear.png"
+      },
+      {
+        "title": "Brakes",
+        "content": [
+          "With the constraint of 1.3 km of test track to work with comes the need to dissipate kinetic energy quickly. Our Eddy Current (EC) brakes work by inducing an eddy current in the web of the aluminum rail, creating opposing magnetic fields that work to brake the pod. They are also designed to counteract the stress produced by these potentially large forces."
```
Comment (Stephenyhlin) [inline routes/flock.json]: "With the constraint of 1.3 km of test track to work with comes the need to dissipate kinetic energy quickly."

Should be changed to 

"Given the constraint of 1.3km of test track, we had to create a braking system to dissipate kinetic energy quickly"

Change "creating opposing magnetic fields that work to brake the pod." to 
"creating opposing magnetic fields that slow the pod to a stop"

File: routes/flock.json [hunk 4]
```diff
@@ -119,43 +119,282 @@
...
+        "title": "Frame",
+        "content": [
+          "Similar to Goose II, we chose to use the ladder frame design with 2 L-beams connected by 4 I-beams. Improvements in the new frame design include a longer frame for the friction drive propulsion system and Eddy Current brakes, and a narrower frame to simplify component mounting and reduce torsional forces."
+        ],
+        "parallax": "/images/flock/goose3/frame-skeleton.png"
+      },
+      {
+        "title": "Electrical",
+        "content": [
+          "The nervous system of our pod, the electrical system connects the 24V and 567V powercells to the primitive sensors and motors installed throughout the pod. Arduino boards and Raspberry Pi's act as central hubs that interpret sensory data and control the flow of energy into each primitive module."
```
Comment (Stephenyhlin) [inline routes/flock.json]: Get rid of "The nervous system of our pod".

Just start the sentence as "The electrical system"

File: routes/flock.json [hunk 5]
```diff
@@ -119,43 +119,282 @@
...
+          "The prototype is about 0.4m (15.75\") wide, 2.0m (6.56') long, 0.4m (15.75\") tall, around 81.7 kg in weight, and is designed to reach a safe and slightly faster speed of 44m/s."
+        ]
+      },
+      {
+        "title": "Linear Induction Motor",
+        "content": [
+          "A linear induction motor produces thrust through a linear force rather than through torque. This allows it to function on an unpowered conductive track, thereby saving costs in maintenance and per-unit track costs. Despite the disadvantage of end effects (or \"turbulence\") leading to lower speeds, the LIM offers contactless propulsion on a simpler track such as an aluminum one. Furthermore, the LIM allows for a centralized design that reduces the weight of the entire pod, and there are a variety of opportunities to scale up the LIM design and promises of better levitation technologies to use with the LIM."
```
Comment (Stephenyhlin) [inline routes/flock.json]: "and there are a variety of opportunities to scale up the LIM design and promises of better levitation technologies to use with the LIM."

to something like 

"allowing for more opportunities to scale up and incorporate better levitation technologies into the LIM "

File: routes/flock.json [hunk 6]
```diff
@@ -119,43 +119,282 @@
...
+        "title": "Shell",
+        "content": [
+          "Designed with the focus on aesthetics, mass and aerodynamics, and inspired by the F-117 bomber, our smoked polycarbonate shell covers the entire pod while still allowing for its individual components to be shown and easily tweaked. Our GOOSE IV shell was also designed to stand out, with ample space to display our logo as well as promotional material."
+        ],
+        "parallax": "/images/flock/goose4/g4-shell.png"
+      },
+      {
+        "title": "Guidance",
+        "content": [
+          "Guiding the pod along the track, the guidance system provides lateral stability (so the pod doesn't tilt to one side) and controls vertical suspension. This system is split into the front and back of the pod, with the front holding the pod up and absorbing shocks from the track, and the back preventing the pod from derailing."
```
Comment (Stephenyhlin) [inline routes/flock.json:258]: Get rid of the "Guiding the pod along the track"

keep the sentence as "The guidance system ..."

File: routes/flock.json [hunk 7]
```diff
@@ -119,43 +119,282 @@
...
+        "content": [
+          "The power system comprises of 2 parts: High Voltage system (HV), and Low Voltage system (LV). The HV system powers the linear induction motor, while the LV system powers all other components on the pod. An enclosure is used to insulate the batteries used in the 2 systems, and a Battery Management system ensures that consistent and stable power is provided to the pod."
+        ],
+        "parallax": "/images/flock/goose4/power.png"
+      },
+      {
+        "title": "Motor Control and Inverter",
         "content": [
-          "This is GOOSE III pod, the one that we are building in the next few terms to get ready for Competition III."
+          "The primary way of controlling the operating speed of the Linear Induction Motor (LIM) is to use an inverter to adjust the AC output frequency to the LIM using the DC battery power supply. The controller is used to ensure a stable power output from the non-constant three-phase AC loead from the LIM, while also accounting for the end effects (or \"turbulence\") produced by the LIM."
```
Comment (Stephenyhlin) [inline routes/flock.json:272]: you have a typo. I think the word "loead" should be "load"?

File: routes/flock.json [hunk 8]
```diff
@@ -119,43 +119,282 @@
...
         ]
       }
     ]
   },
   {
     "name": "Goose V",
-    "thumbnail": "/images/flock/goosex.png",
+    "thumbnail": "/images/flock/goose5.png",
     "url": "goose_v",
+    "desc": "UnLIMited potential.",
```
Comment (Stephenyhlin) [inline routes/flock.json:288]: Is this on purpose? Because of the LIM stuff from goose 4?

If not, can you put it back to Unlimited. TBH, i would just leave it as Unlimited, because I feel like many people will. just click this tab without reading goose 4

File: routes/flock.json [hunk 9]
```diff
@@ -119,43 +119,282 @@
...
+          "The prototype is about 0.533m (21.68\") wide, 2.296m (7.5') long, 0.356m (14.76\") tall, around 82 kg in weight, and is designed to reach a safe and faster speed of 44m/s."
+        ]
+      },
+      {
+        "title": "Linear Induction Motor",
+        "content": [
+          "Like a Rotary Induction Motor but cut up on its centre axis and with the stator and rotor unrolled flat, a linear induction motor produces thrust through a linear force rather than through torque. This allows it to function on an unpowered conductive track, thereby saving costs in maintenance and per-unit track costs. Despite the disadvantage of end effects (or \"turbulence\") leading to lower speeds, the LIM offers contactless propulsion on a simpler track such as an aluminum one. Furthermore, the LIM allows for a centralized design that reduces the weight of the entire pod, and there are a variety of opportunities to scale up the LIM design and promises of better levitation technologies to use with the LIM. We use the DLIM (or Double-sided LIM) design, which involves 2 LIMs on either side of the I-beam track, to allow for a stronger magnetic field to be maintained."
```
Comment (Stephenyhlin) [inline routes/flock.json:301]: "Like a Rotary Induction Motor but cut up on its centre axis and with the stator and rotor unrolled flat"

is too wordy. Change it to 

"Similar to a Rotary Induction Motor, a linear induction motor produces thrust through a linear force."

Comment (Stephenyhlin) [reply routes/flock.json:301]: And change the a linear induction motor to a Linear Induction Motor

Comment (Stephenyhlin) [reply routes/flock.json:301]: Also, i noticed you copied a lot from Goose 4. Whatever changes i suggested to thos sections, please implement it here.

File: routes/flock.json [hunk 10]
```diff
@@ -119,43 +119,282 @@
...
+        ],
+        "parallax": "/images/flock/goose5/DLIM_Render.png"
+      },
+      {
+        "title": "Frame",
+        "content": [
+          "The main structural component of the pod, the frame houses all of the pod's mechanical and electrical subsytems. Made from 6061-T6 square aluminum tubing, the frame is welded in a truss structure and weighs 10 kg. The material and construction of the frame is designed to withstand all acceleration, braking, vibration and structural loads during the run."
```
Comment (Stephenyhlin) [inline routes/flock.json:308]: Get rid of the first sentence. Unneeded

File: routes/flock.json [hunk 11]
```diff
@@ -119,43 +119,282 @@
...
+        ],
+        "parallax": "/images/flock/goose5/FRAME_DESIGN .JPG"
+      },
+      {
+        "title": "Guidance",
+        "content": [
+          "The guidance system maintains lateral and horizontal stability. It accounts for any variations or imperfections in the track that may induce vibrations and bump impulses to the vehicle, and ensures the pod stays on track without damage. The guidance system also ensures that the pod maintains consistent contact with the track, as the LIM and sensors are affected by even the smallest of deviations.",
```
Comment (Stephenyhlin) [inline routes/flock.json]: Change "and ensures the pod stays on track without damage"

to "ensuring the pod stays on track without damage"

File: routes/flock.json [hunk 12]
```diff
@@ -119,43 +119,282 @@
...
+        "title": "Battery and Battery Management",
+        "content": [
+          "The battery management system (BMS) is used to monitor and balance the cell array used to power the Linear Induction Motors and low voltage systems. The lithium ion cells experience multiple charge and discharge cycles, thus they will degrade if improperly managed without the BMS. The BMS also utilizes passive cell balancing to ensure that all batteries in the stack are evenly charged, thereby prolonging cell life and maintaining healthy charge levels."
+        ],
+        "parallax": "/images/flock/goose5/battery.png"
+      },
+      {
+        "title": "Embedded Sensors",
+        "content": [
+          "The pod consists of an array of 32 sensors, with 11 unique models. A few examples of models would be the Piezo, which measures vibrations both at the wheel and frame level, and the Phototransitor that reads the RPM of the pod's wheels for calculating speed."
```
Comment (Stephenyhlin) [inline routes/flock.json]: Phototransitor

change that to 

Phototransistor

File: routes/flock.json [hunk 13]
```diff
@@ -119,43 +119,282 @@
...
+        ]
+      },
+      {
+        "title": "Physical System Architecture: Communications",
+        "content": [
+          "There are 2 major components comprising the communications system:",
+          "<ul><li>The Networking and TCP component utilizes a Desktop Raspberry Pi. It facilitates relaying sensor data from the sensors to the desktop dashboard to be displayed, and to send commands from the remote desktop dashboard back to the pod.</li>",
```
Comment (Stephenyhlin) [inline routes/flock.json]: facilitates relaying sensor data 

to 

it relays sensor data

Comment (Stephenyhlin) [reply routes/flock.json]: to send commands from the remote desktop dashboard back to the pod.

to 

sends commands from the remote desktop dashboard back to the pod

File: routes/flock.json [hunk 14]
```diff
@@ -119,43 +119,282 @@
...
+          "<li>The CAN Bus connects a Raspberry Pi with Arduinos/Sensors. It serves as a cost-effective way of transmitting data between the devices with minimal interference from other subsystems in the pod.</li></ul>"
+        ]
+      },
+      {
+        "title": "User Interface: Dashboard",
+        "content": [
+          "Developed using QT Designer, QML and C++, the desktop system's dashboard displays sensor data about the pod's performance and provides a platform to control the pod remotely. The dashboard is organized in containers, where each container displays information regarding similar sensor data in an easily interpreted fashion. Gauges show the danger levels associated with a reading, and are thus used extensively. The colour warnings and indicators are designed to easily stand out to the user, thereby warning them to take action more effectively.",
```
Comment (Stephenyhlin) [inline routes/flock.json]: are thus used extensively

remove the thus

File: routes/flock.json [hunk 15]
```diff
@@ -119,43 +119,282 @@
...
+      }
+    ]
+  },
+  {
+    "name": "Goose X",
+    "thumbnail": "/images/flock/goosex.png",
+    "url": "goose_x",
     "desc": "",
-    "banner": "/images/flock/goose3/torontorender.jpg",
+    "banner": "/images/flock/goosex/torontorender.jpg",
```
Comment (Stephenyhlin) [inline routes/flock.json:378]: Why are you changing goose3 to gooseX?

Comment (jmawloo) [reply routes/flock.json:378]: The image is supposed to be used for GooseX to begin with.
